bam.iobio/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@overture-stack/bam-visualizations",
+	"version": "0.1.0",
+	"private": true,
+	"type": "module",
+	"dependencies": {
+		"@emotion/react": "^11.11.4",
+		"bam.iobio": "file:../../bam.iobio.io.poc",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
+		"react-scripts": "5.0.1"
+	},
+	"scripts": {
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"eject": "react-scripts eject"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/overture-stack/iobio-components"
+	},
+	"browserslist": {
+		"production": [">0.2%", "not dead", "not op_mini all"],
+		"development": ["last 1 chrome version", "last 1 firefox version", "last 1 safari version"]
+	},
```

Comment:
```suggestion
```

Reply:
FYI it looks like react-scripts actually require this

![Screenshot 2024-06-06 at 10 49 22 AM](https://github.com/overture-stack/iobio-components/assets/15621876/3af2d484-b806-4ac6-84e9-6b80d691b8d1)

Reply:
Added back: https://github.com/overture-stack/iobio-components/pull/15/commits/ffc8d7afab5d0a89e640f170f1b47af4ab8edcc5

Reply:
ah! nice find. thankfully, this should only be necessary for the demo CRA thing, but ideally, it wouldn't be for the wrapper/component itself, right? those are just TSX files, which babel transpiles into `.js` directly (no CRA involved).